polaris/polaris/integrations/transactions.py

```diff
@@ -45,6 +53,7 @@ def poll_pending_deposits(cls, pending_deposits: QuerySet
         memory usage.
 
         :param pending_deposits: a django Queryset for pending Transactions
+        :return: a list of Transaction database objects to execute
```

Comment:
`a list of Transaction database objects to execute` feels confusing, it sounds like its going to execute a database transaction, it should be explicit that its Transaction entries for transactions which have successful user deposits and are ready to be executed on the stellar network